Adapters/Vungle/ISVungleAdapter/src/main/java/com/ironsource/adapters/vungle/banner/VungleBannerAdapter.kt

```diff
@@ -209,6 +214,25 @@ class VungleBannerAdapter(adapter: VungleAdapter) :
 
             else -> null
         }
+
+        try {
+            if (bannerSize.isAdaptive && vungleAdSize != null) {
+                val context = ContextProvider.getInstance().applicationContext
+                val adaptiveSize = VungleAdSize.getAdSizeWithWidth(context,
```

Comment:
we r not suppose to use this. This api has some inconsistencies.
we should use
VungleAdSize.getAdSizeWithWidthAndHeight(width: bannerSize.containerParams.width, height: 0);